docusaurus.config.js

```diff
@@ -85,7 +85,8 @@ const config = {
                     items: [
                         {
                             label: versions[0],
-                            to: `docs/${versions[0]}/introduction`,
+                            to: `docs/${versions[0]}/intro/about`,
+
                         },
                         ...versions.slice(1).map((version) => ({
```

Comment:
NIT: did we want to show all history docs in our website?

Reply:
We can add a limit if the history is long.